docs/proxy/atlas_proxy.md

```diff
@@ -6,7 +6,7 @@ In order to make the Atlas-Amundsen integration smooth, we've developed a python
 Usage and Installation of `amundsenatlastypes` can be found [here](https://github.com/dwarszawski/amundsen-atlas-types/blob/master/README.md)
 
 Minimum Requirements:
-- amundsenatlastypes==1.1.3
+- amundsenatlastypes==1.1.4
```

Comment:
@mgorsk1 since we are on to this, can you please specify the min version of Atlas required for the Amundsen.

Reply:
https://github.com/amundsen-io/amundsen/blob/master/docs/faq.md it's available here. I will raise separate MR to update amundsen-atlas-types there